src/admin-views/privacy.php

```diff
@@ -22,16 +22,31 @@
 
 	<p class="privacy-policy-tutorial"><?php esc_html_e( 'Through the usage of Event Tickets, Event Tickets Plus, and Community Tickets, information may be collected and stored within your website’s database.', 'event-tickets' ); ?></p>
 	<p class="privacy-policy-tutorial"><strong><?php esc_html_e( 'Suggested text:', 'event-tickets' ); ?></strong></p>
-	<p><?php esc_html_e( 'If you create, submit, import, save, or publish event ticket information, as well as RSVP or purchase tickets to events, such information is retained in the local database:', 'event-tickets' ); ?></p>
+	<p><?php printf(
+			esc_html__( 'If you create, submit, import, save, or publish event %1$s information, as well as obtain %2$s or purchase %3$s to events, such information is retained in the local database:', 'event-tickets' ),
```

Comment:
This only escapes the __ string, not the strings you are inserting via printf(). Can you follow the other examples where I've made suggestions that does it like echo esc_html( sprintf( .... ) ); here elsewhere in your PR?